mesonbuild/compilers/mixins/clang.py

```diff
@@ -42,9 +42,10 @@
 }  # type: T.Dict[str, T.List[str]]
 
 class ClangCompiler(GnuLikeCompiler):
-    def __init__(self):
+    def __init__(self, defines: T.Dict[str, str]):
```

Comment:
```suggestion
    def __init__(self, defines: T.Optional[T.Dict[str, str]]):
```

Since this can be `None`, right?

Reply:
Yes, this looks better, thanks.